cibuild.cmd

```diff
@@ -145,5 +146,11 @@ exit /b 1
 @REM Kill any instances of msbuild.exe to ensure that we never reuse nodes (e.g. if a non-roslyn CI run
 @REM left some floating around).
 
-taskkill /F /IM vbcscompiler.exe 2> nul
-taskkill /F /IM msbuild.exe 2> nul
+@REM An error-level of 1 means that the process was found, but could not be killed.
+taskkill /F /IM vbcscompiler.exe > nul
+if %ERRORLEVEL% == 1 exit /b 1
+
+taskkill /F /IM msbuild.exe > nul
```

Comment:
As long as we're changing this code we should kill msbuild first, then vbcscompiler.  Otherwise there is a subtle change msbuild could respawn a new vbcscompiler.